packages/components/src/local/adjudicate-turn-form/index.tsx

```diff
@@ -95,6 +95,10 @@ export const AdjudicateTurnForm: React.FC<PropTypes> = ({ formHeader, formData,
         platformType={icon.platformType}
       >
         {formHeader}
+        { plansSubmitted &&
+         <h5 className='sub-title'>(Form disabled, plans submitted)</h5>
```

Comment:
Why an `h5` here just out of curiosity?

Reply:
It was to make the text smaller.

Reply:
So `h1`, `h2`, `h3` etc.... have nothing to do with font size. That's a common misconception. They are there to represent the hierarchy. So an `h1` for example is the most important part of that section/page and a `h5` is not very important at all.  See this article for more information: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

I'd recommend choosing a more appropriate header level and then using CSS to change the font-size.

Reply:
Ok, here's my thinking.  I don't know why, but the `formHeader` is an `h2`.  I don't mind actual size, but I wanted it less dominant than `h2`  - so `h5` worked. Here's what `inspect` shows:
![image](https://user-images.githubusercontent.com/1108513/95759090-2bdb6a80-0ca1-11eb-8910-2f67d52f1a2f.png)

Reply:
In that case, a `h3` would be appropriate :)